book/src/cross-compiling.md

```diff
@@ -0,0 +1,41 @@
+# Cross-compiling
+
+Lighthouse supports cross-compiling, allowing users to run a binary one
+platform (e.g., `aarch64`) that was compiled on another platform (e.g.,
+`x86_64`).
+
+
+## Instructions
+
+Cross-compiling requires [`Docker`](https://docs.docker.com/engine/install/),
+[`rustembedded/cross`](https://github.com/rust-embedded/cross) and for the
+current user to be in the `docker` group.
+
+The binaries will be created in the `target/` directory of the Lighthouse
+project.
+
+### Targets
+
+The `Makefile` in the project contains four targets for cross-compiling:
+
+- `build-x86_64`: builds an optimized version for x86_64 processors (suitable
```

Comment:
I'm surprised that this produces a portable binary, but it seems to (I ran the v0.2.2 release on my laptop just fine). The binary must still be compiled with `-march=native` inside the Docker image, which would tailor it to the host machine, so I'm not sure if we just got lucky with the choice of host. I've opened https://github.com/sigp/lighthouse/issues/1505 to track this.